catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/LatestCommand.java

```diff
@@ -48,7 +49,7 @@
 
     private static final String NUMBER = "#";
 
-    @Argument(name = "NUMBER_OF_ITEMS", description = "Number of maximum records to display. 0 returns all records.", index = 0, multiValued = false, required = false)
+    @Argument(name = "NUMBER_OF_ITEMS", description = "Number of maximum records to display. 0 returns 1000 records.", index = 0, multiValued = false, required = false)
```

Comment:
Why not just default to 1000 and eliminate the magical 0 logic? It would let you get rid of the ternary below as well. Same goes for both commands. If it's for the sake of backwards-compatibility, the behavior is already changing, right?

Reply:
Well... I think @kcwire has mentioned commands are technically a public api.. so maybe we should do a best effort backwards compatibility and treat 0 as int.max_value? Honestly i'm fine either way, because right now they are limited to 1000 results without this fix anyways. Do you have a preference @brendan-hofmann ?